llvm/test/MC/RISCV/rv32zaamo-invalid.s

```diff
@@ -0,0 +1,11 @@
+# RUN: not llvm-mc -triple riscv32 -mattr=+experimental-zaamo < %s 2>&1 | FileCheck %s
```

Comment:
Can we split `rv32a-invalid.s` into two files? I think it is better than duplicating tests for new extensions.

Reply:
This can be done in the future, I think.
Current implementation refers to the `Zmmul` (which is a sub-extension of M extension). The case is the same for `Zaamo` and `Zalrsc` I think.